frontend/rust-lib/flowy-document2/src/event_handler.rs

```diff
@@ -309,16 +311,46 @@ impl From<(&Vec<BlockEvent>, bool)> for DocEventPB {
   }
 }
 
-/**
-* Handler for converting a document to a JSON string, HTML string, or plain text string.
-
-* @param data: AFPluginData<[ConvertDocumentPayloadPB]>
-
-* @param manager: AFPluginState<Weak<DocumentManager>>
-
-* @return DataResult<[ConvertDocumentResponsePB], FlowyError>
- */
-pub async fn convert_document(
+/// Handler for converting a document to a JSON string, HTML string, or plain text string.
+///
+/// ConvertDocumentPayloadPB is the input of this event.
+/// ConvertDocumentResponsePB is the output of this event.
+///
+/// # Examples
+///
+/// Basic usage:
+///
+/// ```txt
+/// // document: [{ "block_id": "1", "type": "paragraph", "data": {"delta": [{ "insert": "Hello World!" }] } }, { "block_id": "2", "type": "paragraph", "data": {"delta": [{ "insert": "Hello World!" }] }
+/// let test = DocumentEventTest::new().await;
+/// let view = test.create_document().await;
+/// let payload = ConvertDocumentPayloadPB {
+///   document_id: view.id,
+///   range: Some(RangePB {
+///     start: SelectionPB {
+///       block_id: "1".to_string(),
+///       index: 0,
+///       length: 5,
+///     },
+///     end: SelectionPB {
+///       block_id: "2".to_string(),
+///       index: 5,
+///       length: 7,
+///     }
+///   }),
+///   export_types: ConvertTypePB {
```

Comment:
what is the ConvertTypePB

Reply:
It's ParseTypePB, i forgot to update this